lang/main.json

```diff
@@ -490,6 +492,7 @@
         "invitedOneMember": "{{name}} has been invited",
         "invitedThreePlusMembers": "{{name}} and {{count}} others have been invited",
         "invitedTwoMembers": "{{first}} and {{second}} have been invited",
+        "canJoinConference": "You can now join meeting {{room}}. ",
```

Comment:
```suggestion
        "canJoinConference": "You can now join the meeting {{room}}. ",
```

Reply:
This was deleted, because we no longer show that notification.